ipatests/test_integration/test_uninstallation.py

```diff
@@ -119,3 +119,20 @@ def test_failed_uninstall(self):
             self.master.run_command([
                 paths.DSCTL, serverid, 'remove', '--do-it'
             ])
+
+
+class TestUninstallWithoutDNS(IntegrationTest):
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master, setup_dns=False)
+
+    def test_uninstall_server_without_dns(self):
+        """Test if server setup without dns uninstall properly
+
+        IPA server uninstall was failing if dns was not setup.
+        This test check if it uninstalls propelry.
```

Comment:
There is a typo in propelry / properly.
No need to re-run the temp commit, you can fix the typo and remove the temp commit and I will ACK when PRCI completes.